stylesheet/main.css

```diff
@@ -107,41 +124,31 @@ h2 {
     color: transparent;
 }
 
-div paperclip{position: absolute;
+.paperclip{position: absolute;
              width :50px;
              height:50px;
-            top:10px;
-            right-top: 7px;
+            top: -10px;
+            right: -7px;

```

Comment:
Incorrect position values.  Use negative values to move top and right positioned elements up and to the right, respectively